styles/microsoft-word/catppuccin.user.css

```diff
@@ -0,0 +1,358 @@
+/* ==UserStyle==
+@name Microsoft Word Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/microsoft-word
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/microsoft-word
+@version 0.0.1
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/microsoft-word/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Amicrosoft-word
+@description Soothing pastel theme for Microsoft Word
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select flavour "Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+
+==/UserStyle== */
+
+@-moz-document domain('live.com') {
```

Comment:
The domain is live.com here but in userstyles.yml it is office.com. Not sure which is the right one but we should just have a single URL.

Reply:
solved in discord discussion